packages/core/src/ShadowEffect/ShadowEffect.spec.js

```diff
@@ -9,7 +9,7 @@ describe('ShadowEffect', () => {
     const mockOnClick = jest.fn()
     render(
       <ShadowEffect onOpen={mockOnOpen} data-testid='overlay'>
-        <Input onClick={mockOnClick} data-testid='input' />
+        <Input onClick={mockOnClick} id='input' data-testid='input' />
```

Comment:
also just added some ids here to clean up warnings when running the test suite

Reply:
Thanks for improving my tests